shared/bitutil/bit.go

```diff
@@ -27,21 +27,16 @@ func SetBitfield(index int, committeeLength int) []byte {
 
 // CheckBit checks if a bit in a bit field (small endian) is one.
 func CheckBit(bitfield []byte, index int) (bool, error) {
-	chunkLocation := (index + 1) / 8
-	indexLocation := (index + 1) % 8
-
-	if indexLocation == 0 {
-		indexLocation = 8
-	} else {
-		chunkLocation++
+	if len(bitfield) < (index+7)>>3 {
```

Comment:
could you again make this a var `(index+7)>>3`, its hard to undestand what this expression is if I have no context of the function

Reply:
changed to mathutil.CeilDiv8(index)